src/doubleTrouble.js

```diff
@@ -25,6 +25,13 @@
  */
 function doubleTrouble(array, target) {
   // write code here
+  return array.reduce(
+    (resultArray, value) => {
+      if (resultArray[resultArray.length - 1] + value !== target) {
+        resultArray.push(value);
```

Comment:
Will be better to not mutate an array
```suggestion
        return [... resultArray, value];
```
But your solution is good

Reply:
understood. But, please, leave the comments with implemetation why!..